smac/smbo/local_search.py

```diff
@@ -16,7 +16,7 @@
 class LocalSearch(object):
 
     def __init__(self, acquisition_function, config_space,
-                 epsilon=0.000001, max_iterations=None, rng=None):
+                 epsilon=0.00001, max_iterations=None, rng=None):
```

Comment:
This is an increase in epsilon, while the PR comment states that this PR lowers epsilon.